user-docs/pupil-player.md

```diff
@@ -198,17 +198,45 @@ These plugins are simple unique plugins, that operate on the gaze data for analy
 
 > {{< webp-img src="/images/pupil-player/plugin/offline-surface-tracker.webp" alt="Offline Surface Tracker" >}}
 
-This plugin is an offline version of the [Surface Tracking](#surface-tracking) plugin for Pupil Capture. You can use this plugin to detect markers in the recording, define surfaces, edit surfaces, and create and export visualizations of gaze data within the defined surfaces.
+This plugin is an offline version of the [Surface Tracking](#surface-tracking) plugin for Pupil Capture. 
+You can use this plugin to detect markers in the recording, define surfaces, edit surfaces, and create and export visualizations of gaze data within the defined surfaces.
 
 Here is an example workflow for using the `Offline Surface Detector` plugin to generate heatmap visualizations and export surface data reports:
 
   + Load `Offline Surface Detector` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.
   + Add surface - if you do not have any defined surfaces, you can click on the `Add surface` button when the markers you want to user are visible or just click the circular `A` button in the left hand side of the screen.
-  + Surface name and size - In the `Marker Detector` GUI window, define the surface name and real world size. *Note* - defining size is important as it will affect how heatmaps are rendered.
+  + Surface name and size - In the `Marker Detector` GUI window, define the surface name and real world size. 
+  *Note* - defining size is important as it will affect how heatmaps are rendered.
   + Set trim marks - optional, but if you want to export data for a specific range, then you should set the trim marks.
-  + Recalculate gaze distributions - click the `(Re)calculate gaze distributions` button after specifying surface sizes. You should now see heatmaps in the Player window (if gaze positions were within your defined surfaces).
+  + Recalculate gaze distributions - click the `(Re)calculate gaze distributions` button after specifying surface sizes. 
+  You should now see heatmaps in the Player window (if gaze positions were within your defined surfaces).
   + Export gaze and surface data - click `e` and all surface metrics reports will be exported and saved for your trim section within your `export` folder.
 
+All files generated by the `Offline Surface Detector` will be located in the subfolder `surfaces`. 
+The different reported metrics are:
+
+  + `surface_visibility.csv` - Overview of how many world camera frames each surface was contained in.
+  + `surface_gaze_distribution.csv` - Overview of how many gaze samples have been collected on each individual surface and outside of surfaces.
+  + `surface_events.csv` - List of gaze-enter and gaze-exit events for all surfaces.
```

Comment:
@marc-tonsen  this is actually not gaze ent and gaze exit but surface enter and exit events.